dev/io.openliberty.jakartaee10.internal_fat/fat/src/io/openliberty/jakartaee10/internal/apps/jakartaee10/web/persistence.xml

```diff
@@ -0,0 +1,11 @@
+<persistence xmlns="https://jakarta.ee/xml/ns/persistence"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://jakarta.ee/xml/ns/persistence https://jakarta.ee/xml/ns/persistence/persistence_3_1.xsd"
```

Comment:
My understanding is there is not a 3.1 updated xsd file for Jakarta Persistence 3.1.